Microsoft.Toolkit.Uwp.UI.Media/Helpers/SurfaceLoader.cs

```diff
@@ -57,7 +57,7 @@ public static async Task<CompositionSurfaceBrush> LoadImageAsync(Uri uri, DpiMod
                 }
 
                 // Load the image
-                CompositionSurfaceBrush brush;
+                CompositionBrush brush;
```

Comment:
This is just a more liberal/base version of it, eh?

Reply:
Yes it's the base class, which is the one we need anyway. I made this change in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3298/commits/bec2112710488a56b35e1b874c7be35b8cc61f68 to remove one unnecessary async state machine, and I needed the returned type to be the correct one as I'm returning a `ValueTask<T>`, which not being an interface is not covariant in `T`. Basically without this change I'd have had to add another `await` to get the result and upcast it, while this way I can just return the task directly, which is more efficient.